src/commands/developer/userdata.js

```diff
@@ -296,6 +296,12 @@ module.exports = {
     async removeUserdata(client, reply, message, argu, locale) {
         const userId = argu[1]
         const sql = `DELETE FROM users WHERE user_id=$userID`
+        const updateConfig = `UPDATE guild_configurations SET setByUserId=$bot, updated_at = CURRENT_TIMESTAMP WHERE setByUserId=$userID`
```

Comment:
I didn't quite understand this part😂
What does guild config has to do with user data deletion .. is it marking that there is an activity from the said user?

Reply:
User id is tied to that data so if the user is deleted, the id needs to be updated to not break configurations

Reply:
Oh! yeah smart call. Its fine thenn